### ISSUE TITLE
How to get/set config options which are from a dict

### ISSUE THREAD

**Issue by surak** [singularityhub/singularity-hpc#561]
When setting the system I can check for a multi-line config option, such as 

```bash
$ shpc config get wrapper_scripts
wrapper_scripts                ordereddict([('enabled', False), ('docker', 'docker.sh'), ('podman', 'docker.sh'), ('singularity', 'singularity.sh'), ('templates', None)])
```

But how can I set one of them back with `shpc config --global set`? The documentation doesn't mention this case.
This is for automated installations where the default configuration file comes from a git clone and only the relevant changes are made.

Any reason why the wrapper is disabled by default, by the way? One clearly needs that for slurm...

**Comment by vsoch** [comment]
Ah! So the normal way (which I agree we need to document better) is to add a colon to separate levels

```
$ shpc config get wrapper_scripts:enabled
wrapper_scripts:enabled        False
```
But there is also a bug in the final step here that writes "is unset" if "not value" and it needs to just check if the value is None (and not false!) Set also has a bug - it was written before we had the nested values.

@surak I'll have a PR with fixes for the above and better docs this evening! And I do think we could have the default be wrapper scripts - I was actually thinking this yesterday! @marcodelapierre and @muffato would that make sense to you as well?

**Comment by muffato** [comment]
Hi @vsoch . In my world, yes, the wrapper scripts should be default

**Comment by vsoch** [comment]
okay here is a start! https://github.com/singularityhub/singularity-hpc/pull/562

I am likely to add a few more test cases tonight (e.g., derivates of true/True false/False null/Null/None/none and the like, but the basic updates should be there!

**Comment by vsoch** [comment]
Scratch that - added them now! :)

**Comment by marcodelapierre** [comment]
Yep, here at Pawsey we also use wrapper scripts as default in our deployments.

**Comment by vsoch** [comment]
Fixed with #562 - will be released shortly.
